src/components/UserStats/MyDashboard.vue

```diff
@@ -33,11 +33,9 @@
         </UserStatsItem>
         <UserStatsItem
           :is-loading="stats.isLoadingStats"
+          :stats-value="stats.collectedAmount"
           :label-text="$t('nft_portfolio_page_state_value')"
         >
-          <template #custom-value>
```

Comment:
Why is this removed?

Reply:
I just formatted the `collectedAmount` in props ( also `createdTotalSales`), so there is no need to do the format here.